packages/server-wallet/e2e-testing/scripts/load-generator.ts

```diff
@@ -24,91 +24,116 @@ const ETH_ASSET_HOLDER_ADDRESS = '0x9eD274314f0fB37837346C425D3cF28d89ca9599';
-    numberOfChannels,
+    createRate,
     prettyOutput,
     roleFile,
-    closeTime,
-    createTime,
-    updateTime,
     outputFile,
+    createWait,
+    closeRate,
+    duration,
   } = await yargs(hideBin(process.argv))
-    .option('closeChannels', {
-      alias: 'c',
-      default: true,
-      type: 'boolean',
-      describe: 'Whether channels are closed',
-    })
     .option('prettyOutput', {
       default: true,
       type: 'boolean',
-      describe: 'Whether the output is formatted nicely with spaces',
-    })
-    .option('numberOfChannels', {
-      alias: 'n',
-      default: 10,
-      describe: 'The amount of channels that should be created.',
-    })
-    .option('createTime', {
-      type: 'number',
-      description: `The range of time for a CreateChannelStep.
-      CreateChannelSteps will be assigned a timestamp randomly from 0 to createTime`,
-      default: ms('10s'),
+      describe: 'Whether the output is formatted nicely with spaces.',
     })
-    .option('updateTime', {
-      type: 'number',
-      description: `The range of time for a UpdateChannelStep.
-      Each UpdateChannelStep will be incremented by a random value from 0 to updateTime `,
-      default: ms('10s'),
-    })
-    .option('closeTime', {
-      type: 'number',
-      description: `The range of time for a CloseChannelStep.
-      CloseChannelSteps will be assigned a timestamp randomly from updateTime to CloseTime`,
-      default: ms('10s'),
-    })
-    .option('amountOfUpdates', {
-      alias: 'u',
-      description:
-        'The amount of updates that should be done per channel. If 0 no updates are made.',
-      default: 0,
+    .option('outputFile', {
+      alias: 'o',
+      description: 'The file to write the generated load to.',
+      default: 'temp/test_load.json',
     })
     .option('roleFile', {
       alias: 'f',
-      describe: 'The path to a file containing the role information',
+      describe: 'The path to a file containing the role information.',
       default: './e2e-testing/test-data/roles.json',
     })
-    .option('outputFile', {
-      alias: 'o',
-      description: 'The file to write the generated load to',
-      default: 'temp/test_load.json',
+    .option('duration', {
+      alias: 'd',
+      default: 30,
+      describe: `The amount of time (in seconds) that the load should run for.
+      This dictactes the max timestamp a step can have.`,
+    })
+    .option('createRate', {
+      alias: 'cr',
+      default: 5,
+      describe: 'The number of channels that should be created per a second.',
+    })
+    .option('createWait', {
+      default: 5,
+      describe: `The minumum amount of time (in seconds) to wait for a channel be fully open, before another step is scheduled.`,
+    })
+    .option('closeRate', {
+      default: 5,
+      describe:
+        'The amount of channels to be closed per a second. If this is larger than the createRate then all channels will eventually get closed.',
```

Comment:
This option seems nice!
```suggestion
        'The amount of channels to be closed per a second. If this is larger than the createRate then all channels will eventually get closed. Otherwise, some channels will remain open.',
```